src/test/java/uk/gov/hmcts/ccd/endpoint/std/GlobalSearchEndpointIT.java

```diff
@@ -205,6 +193,45 @@ public void shouldThrowValidationErrorsWhenDataNull() throws Exception {
             .andReturn();
     }
 
+
+    @Test
+    public void shouldAuditLogSearchCases() throws Exception {
+
+        // ARRANGE
+        stubElasticSearchSearchRequestWillReturn();
```

Comment:
can we extend stub to return at least two cases so that we can check joiner in the audit log as well..

Reply:
fixed in [30d5318](https://github.com/hmcts/ccd-data-store-api/pull/1672/commits/30d5318a911e804922059a74e1d8b590dbdf9b5c)